lua/kubectl/actions/informer.lua

```diff
@@ -60,6 +60,9 @@ local function decode_json_objects(json_strings)
 end
 
 local function process_event(builder, event)
+  if not event.type then
```

Comment:
Did you have an error with event.type here? It shouldn't be happening since all the events that get here are from the kubernetes watch event, could you print out the event?

Reply:
the problem is where you append `?watch=true&resourceVersion=...`
On custom resources, this produces `"404 page not found"`
I found this out after merging the informer changes to my branch.

I don't think `watch` and `resourceVersion` is applicable for any resource, so this is to be able to fallback

Reply:
hmm that could be true, do you have a crd I could test it out with?

Reply:
nvm got it!

Reply:
So what happened was that the crds url was different and that's why we didn't get the events, after that the issue became the difference in tables between crds and normal view. That's been fixed now!